languages/java/highlights.scm

```diff
@@ -212,7 +212,7 @@
   "yield"
 ] @keyword
 
-"new" @operator
+"new" @keyword.operator
```

Comment:
`@keyword.operator` is not a capture supported by Zed themes.

Reply:
Actually it might be wrong to call `new` an operator at all, it's mentioned as a [keyword here](https://docs.oracle.com/javase/specs/jls/se8/html/jls-15.html#jls-15.9):

>  If passed explicitly, the type arguments to the constructor immediately follow the keyword new.